wails-frontend/src/pages/backupKeyPairs.tsx

```diff
@@ -117,6 +117,7 @@ function BackupKeyPairs() {
                 <Clipboard
                   displayedContent={publicKey}
                   rawContent={publicKey as string}
+                  toggleHover={false}
```

Comment:
For what you set `toggleHover` to `false` ?

Reply:
@SlnPons and I  (for multiple product related reasons) have decided to remove the toggle feature, but keep it in the code base. So we only remove it from components but it stays in the ui-kit.

Reply:
> @SlnPons and I (for multiple product related reasons) have decided to remove the toggle feature, but keep it in the code base. So we only remove it from components but it stays in the ui-kit.

OK, but if you decide to set it to Off, that means that there is no default value in the UI-KIT.

Unless you've made sure that when you don't use the 'toggleHover' value, there's no hover.

Reply:
We will do this update in a new PR